src/features/projects/screens/SingleProjectDetails.tsx

```diff
@@ -109,7 +110,7 @@ function SingleProjectDetails({}: Props): ReactElement {
                   ? `?embed=true&callback=${callbackUrl}`
                   : '?embed=true'
               }`
-            : ''
+            : `${isAuthenticated ? (routeContainProfile ? 'profile' : '') : ''}`
```

Comment:
Sorry, but it still does not make sense for me to default the navigation for logged in users to their profile if visiting a project from the homepage and then use the button to navigate back:

https://user-images.githubusercontent.com/1532418/212468061-910ea137-04d1-45ed-a3f5-0e0262e09755.mp4

Reply:
I have removed of mine  previous implementation , now the back button will redirect user only   to`' / `' for user coming from using any route